bugsnag-android-core/src/main/java/com/bugsnag/android/DeviceData.java

```diff
@@ -224,10 +226,9 @@ static long calculateTotalMemory() {
      * Get the usable disk space on internal storage's data directory
      */
     @SuppressLint("UsableSpace")
-    long calculateFreeDisk() {
+    long calculateFreeDisk(File dataDirectory) {
```

Comment:
As discussed - this should use the member variable not a parameter.

Reply:
Updated to use the member variable.